sormas-api/src/main/java/de/symeda/sormas/api/event/EventDto.java

```diff
@@ -66,6 +69,10 @@
 
 	@Required
 	private EventStatus eventStatus;
+	@Required
```

Comment:
Can't be required because it's possible that other countries don't want to use this feature.